web/cobrands/bromley/layout.scss

```diff
@@ -1,135 +1,270 @@
+  background-color: $bromley_blue;
 }
-.ie6, .ie7, .ie8  {
-    .content {
-        margin: 0;
-        border: none;
+
+// Make the powered by FMS logo work in Old IE. FMS fixes this in IE8, but
+// it's broken in IE6 and 7 too.
+.ie6, .ie7, .ie8 {
+  body.twothirdswidthpage,
+  body.fullwidthpage {
+    .container .content footer a.platform-logo {
+      color: #ffffff;
+      background: none;
+      text-indent: 0px;
+      height: auto;
     }
+  }
 }
 
-// As map can scroll and isn't at the top, give it an edge
-#map_box {
-    border: solid 1px #999;
-    top: 1em;
-    right: 0em;
-    margin: 0;
+// This is our custom wrapper for Bromley's header elements
+.bromley-header {
+  // Bromley's logo has a bottom margin to space it out from the A-Z nav, we
+  // can't do that because they're in separate divs, so add a margin here
+  // instead.
+  margin-bottom: 20px;
 }
 
-// Perhaps fix map location (should be in central?)
-.ie6 #map_box, .ie7 #map_box {
-    right: -480px;
+// The reports_list.scss hides the big banner on the sidebar that tells people
+// what to do with the map (click on it). We don't want that, so put it back
+.big-green-banner {
+  display: block;
 }
 
-.ie7 .big-green-banner {
-    right: 0;
+// Center the mobile app buttons
+#footer-mobileapps p,
+#footer-help p {
+  text-align: center;
 }
 
-// OpenLayers navigation always top right
-#fms_pan_zoom {
-    top: 0.5em !important;
+//////////////////////////////////////////////////////////////////////////////
+// Bromley desktop styles
+//////////////////////////////////////////////////////////////////////////////
+
+// Bromley's logo is bigger on big screens so that it appears to have some
+// left padding
+#site-logo,
+body.frontpage #site-logo {
+  float: left;
+  width: 239px;
+  clear: left;
+  margin-bottom: 20px;
+  // Bromley uses an image element inside a link, so it can be centered using
+  // text-align. We use a background image, so this approximates that.
+  background-position: 50% 0%;
 }
 
-// Fix location of aside sidebar
-body.twothirdswidthpage {
-    .content {
-        aside {
-            @include box-shadow(none);
-        }
-        .sticky-sidebar {
-            top: 1em;
-            aside {
-                position: absolute;
-                top: 0;
-            }
-        }
+// Bromley's .top-links section floats to the right on bigger screens
+.top-links {
+    float: right;
+    width: auto;
+}
+
+// Bromley's search form floats to the right too
+.main-search {
+  width: 35%;
+  margin: 10px 0 0 0;
+  float: right;
+  min-width: 287px;
+  clear: right;
+}
+
+// Bromley's A-Z nav gets the background coloured in on biggger screens to
+// appear full-screen
+.a-z-container {
+  background: #647890;
+  padding: 0;
+}
+
+// The link to open the A-Z nav gets hidden on big screens
+.drop-down-nav > li > a {
+  display: none;
+}
+
+// And the rest of the A-Z nav gets show with a horrible !important
```

Comment:
I'm not sure why, but it's necessary to make the nav display in IE8 and below - I took this from Bromley's CSS - I guess it's a quirk with the way they're doing the hide/show stuff :(